homeassistant/components/obihai/sensor.py

```diff
@@ -48,8 +48,6 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         _LOGGER.error("Invalid credentials")
         return
 
-    """Define and register reboot service"""
```

Comment:
The reason why it complained is because the docstring is supposed to be after the line `def _reboot(call):` also docstrings need to end with a period `.`

Reply:
I appreciate the explanation.  I don't usually program in Python, so I'm learning.